### ISSUE TITLE
Factor out unnecessary steps between tearing down/setting up the state

### ISSUE THREAD

**Issue by perezju** [catapult-project/catapult#3498]
There seem to be many operations that do not need to be repeated between each story run; and done only once at the start of a Telemetry run. We should find those and factor them out.

As a concrete example, surely there is no need to check if the browser needs to be installed between stories:

```
[       OK ] background:social:facebook@{'case': 'background', 'group': 'social'} (26183 ms)
[...]
INFO:root:Chose browser: PossibleAndroidBrowser(browser_type=android-chromium)
WARNING:root:Installing /b/c/b/Android_Nexus5_Perf__1_/src/out/Release/apks/ChromePublic.apk on device if needed.
[...]
[ RUN      ] background:tools:gmail@{'case': 'background', 'group': 'tools'}
```

**Comment by nedn** [comment]
I think we should profile how long does this take first before optimizing for it.

**Comment by anniesullie** [comment]
It should be pretty easy to run perf try jobs on various bots that just rip out operations and compare benchmark_duration?

**Comment by nedn** [comment]
This is a change to catapult & we don't support running perf try jobs against Telemetry change AFAIK.
I also think this can be done easily by adding some debug print around those code & run the thing locally.

**Comment by anniesullie** [comment]
Catapult should be supported on perf try jobs: https://cs.chromium.org/chromium/src/tools/perf/core/trybot_command.py?l=86

But it's fine to run locally too

**Comment by nedn** [comment]
Oh, that's cool to run catapult CL from perf try jobs. Do you just make the change in chromium/src/catapult/.. then trigger the try job with chromium/src/tools/perf/... ?

**Comment by anniesullie** [comment]
Yep, but see the arguments/documentation: https://cs.chromium.org/chromium/src/tools/perf/core/trybot_command.py?l=357
